constants.js

```diff
@@ -33,10 +34,23 @@ const checksWhitelist = {
     [issuesLabelEvent]: []
   },
   'oppia': {
-    [openEvent]: [claCheck, changelogCheck, branchCheck, wipCheck, jobCheck],
-    [reopenEvent]: [changelogCheck, branchCheck, wipCheck, jobCheck],
+    [openEvent]: [
+      claCheck,
+      changelogCheck,
+      branchCheck,
+      wipCheck,
+      jobCheck,
+      modelCheck,
+    ],
+    [reopenEvent]: [
+      changelogCheck,
+      branchCheck,
+      wipCheck,
+      jobCheck,
+      modelCheck,
+    ],
     [PRLabelEvent]: [assigneeCheck],
-    [synchronizeEvent]: [mergeConflictCheck, jobCheck],
+    [synchronizeEvent]: [mergeConflictCheck, jobCheck, modelCheck],
```

Comment:
Not sure - but does it make sense to perform the model and job check on push event as well?

Reply:
Also, why on synchronizeEvent?